.github/workflows/push-master.yml

```diff
@@ -0,0 +1,218 @@
+          # gati inputs
+          use-gati: "true"
+          aws-role-arn-gati: ${{ secrets.AWS_OIDC_IAM_ROLE_ARN_GATI }}
+          aws-lambda-url-gati: ${{ secrets.AWS_LAMBDA_URL_GATI }}
+          # golang inputs
+          go-version-file: go.work
+          # goreleaser inputs
+          goreleaser-args: "--nightly --config ./${{ matrix.package }}/.goreleaser.ci.yaml"
+          goreleaser-dist: goreleaser-pro
+          goreleaser-key: ${{ secrets.GORELEASER_KEY }}
+          # zig inputs
+          use-zig: "true"
+          zig-version: "0.11.0"
+          # docker inputs
+          docker-registry: aws
+          docker-image-tag: devel
+      - name: Save GoReleaser artifacts/metadata
+        uses: actions/upload-artifact@v3
+        with:
+          name: goreleaser_${{ matrix.package }}
+          path: |
+            dist/artifacts.json
+            dist/metadata.json
+
+  cicd-build-publish-artifacts-dev:
+    runs-on: ubuntu-latest
+    needs: [cicd-build-publish-artifacts-dev-matrix]
+    if: always()
+    steps:
+      - name: Check results
+        if: needs.cicd-build-publish-artifacts-dev-matrix.result != 'success'
+        run: exit 1
+      - name: Collect metrics
+        if: always()
+        id: collect-gha-metrics
+        uses: smartcontractkit/push-gha-metrics-action@d1618b772a97fd87e6505de97b872ee0b1f1729a # v2.0.2
+        with:
+          basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
+          hostname: ${{ secrets.GRAFANA_CLOUD_HOST }}
+          this-job-name: cicd-build-publish-artifacts-dev
+        continue-on-error: true
+
+  ci-sonarqube:
+    needs: [ci-lint, ci-test]
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: ci-sonarqube
+        uses: smartcontractkit/.github/actions/ci-sonarqube@main
+        with:
+          # grafana inputs
+          metrics-job-name: ci-sonarqube
+          gc-basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
+          gc-host: ${{ secrets.GRAFANA_CLOUD_HOST }}
+          # sonarqube inputs
+          sonar-token: ${{ secrets.SONAR_TOKEN }}
+          sonar-host-url: ${{ secrets.SONAR_HOST_URL }}
+
+  cd-release:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    outputs:
+      changesets_published: ${{ steps.cd-release.outputs.published }}
+      changesets_publishedPackages: ${{ steps.cd-release.outputs.publishedPackages }}
+    steps:
+      - name: cd-release
+        id: cd-release
+        uses: smartcontractkit/.github/actions/cicd-changesets@8720f77f170718126aab659bdd32edd7432556b3 # cicd-changesets@0.1.1
+        with:
+          # general inputs
+          git-user: app-token-issuer-data-feeds[bot]
+          git-email: app-token-issuer-data-feeds[bot]@users.noreply.github.com
+          pnpm-use-cache: false
+          # aws inputs
+          aws-region: ${{ secrets.AWS_REGION }}
+          aws-role-arn: ${{ secrets.AWS_ROLE_ARN_GATI_CHANGESETS }}
```

Comment:
Will need to setup a GATI to allow changesets to push git tags/create releases/etc